docs/NetworkPeeringForScalarDLAuditor.md

```diff
@@ -0,0 +1,62 @@
+# Create network peering for Scalar DL Auditor mode
+
+This document explains how to connect several private networks for ScalarDL Auditor mode. To make ScalarDL Auditor mode work properly, ScalarDL Ledger and ScalarDL Auditor need to connect with each other. 
+
+## Private network you must connect
+
+Depending on your application (client) deployment, you must connect several private networks.
+
+### Connect two private networks
+
+If you deploy your application on the same private network as ScalarDL Ledger or ScalarDL Auditor, you must connect two private networks.
+
+* [ScalarDL Ledger network] <-> [ScalarDL Auditor network]
+
+### Connect three private networks
+
+If you deploy your application (client) on another private network than ScalarDL Ledger or ScalarDL Auditor, you must connect three private networks.
+
+* [ScalarDL Ledger network] <-> [ScalarDL Auditor network]
+* [ScalarDL Ledger network] <-> [Application (client) network]
+* [ScalarDL Auditor network] <-> [Application (client) network]
+
+## Network requirements
+
+### IP address ranges
+
+To avoid conflicting IP addresses between several private networks, you must create several private networks with different IP address ranges.
```

Comment:
```suggestion
To avoid conflicting IP addresses between the private networks, you must have private networks with different IP address ranges. For example:
```